### PR TITLE
glog: add version 0.7.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **glog/0.7.0**

- add versio 0.7.0
   - require C++14
- make libunwind transitive_libs which uses lzma
- update libunwind

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/glog/all/conanfile.py [hunk 1]
```diff
@@ -56,7 +71,18 @@ def requirements(self):
...
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
 
     def build_requirements(self):
         if Version(self.version) >= "0.6.0":
```
Comment (uilianries) [inline recipes/glog/all/conanfile.py:88]:
```suggestion
        if Version(self.version) >= "0.6.0":
```

The minimal CMake version in 0.7 is 3.22, but the current one listed is 3.16

**Review by uilianries — COMMENTED** [review]
Looking good, only a small detail

File: recipes/glog/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/glog/all/conanfile.py]: @uilianries 
Thanks a lot!
Fixed.

**Review by uilianries — APPROVED** [review]
LGTM. THank you!

**Pull request merged**